packages/resp-client/resp-client.0.10.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "Zach Shipko <zachshipko@gmail.com>"
+authors: ["Zach Shipko <zachshipko@gmail.com>"]
+homepage: "https://github.com/zshipko/resp"
+doc: "https://zshipko.github.io/resp"
+license: "ISC"
+dev-repo: "git+https://github.com/zshipko/resp.git"
+bug-reports: "https://github.com/zshipko/resp/issues"
+tags: ["redis" "protocol"]
+
+depends:
+[
+    "ocaml" {>= "4.07.0"}
+    "dune" {build & >= "2.0.0"}
+    "resp" {= version}
```

Comment:
Are the packages in 0.9 also requiring the strict version like here?
```
    "resp" {= version}
```

if not, we should probably modify them to do it

Reply:
Looks like they were less strict: https://github.com/ocaml/opam-repository/blob/e248948afd8b084309f2db5165d753f2b056fc16/packages/resp-mirage/resp-mirage.0.9.1/opam#L13

they do need an upper bound though